### PR TITLE
Fix doctype and add return types. Require at least PHP 7.1

### PR DESCRIPTION
**Pull request by simonschaufi**
Some files were also using windows line endings instead of `\n`

### CODE REVIEW

**Comment by simonschaufi** [comment]
@Richardds ping

**Review by Richardds — APPROVED** [review]
Hi Simon, thank you for your contribution.

File: src/ECBConverter.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function checkFileCache(): void
         if (!file_exists($this->cache_file) || time() - filemtime($this->cache_file) > $this->cache_timeout) {
             $this->reloadExchangeReferences();
             file_put_contents($this->cache_file, serialize($this->exchange_data), LOCK_EX);
-        } elseif (is_null($this->exchange_data)) {
+        } else if (is_null($this->exchange_data)) {
```
Comment (simonschaufi) [inline src/ECBConverter.php:46]: @Richardds  Why did you change this? This is no Javascript, this is Php 😀

Comment (Richardds) [reply src/ECBConverter.php:46]: Ah, you're right. :D Although [both are valid](https://www.php.net/manual/en/control-structures.elseif.php).

https://github.com/Richardds/ECB-API/commit/e0877e691de3d2a72cfa309f782fa8c0ab0c75c1
